src/licensedcode/data/rules/lgpl-2.1_with_broadcom-linking-exception-2.0_1.RULE

```diff
@@ -3,7 +3,11 @@ license_expression: lgpl-2.1 WITH broadcom-linking-exception-2.0
 is_license_notice: yes
 referenced_filenames:
     - COPYING
-notes: found in the freepascal compiler
+notes: |
+    found in the freepascal compiler
+    Added in SPDX license list 3.26
+    Replaced by license: independent-modules-exception.LICENSE
+is_deprecated: yes
```

Comment:
Why would deprecate this? This rule as it is does not exist anywhere else, does it? Also what happened to the "broadcom-linking-exception-2.0" ref here? this may have been a bug.


So IMHO should effectively keep this as deprecated, but also copy this to a new rule that would be:
`lgpl-2.0-plus WITH independent-modules-exception`

Why lgpl-2.0-plus? Because of the version-less reference to `Library GNU General Public License` .... `library` is 2.0, `lesser` 2.1 and 3.0
Or we use 2.1 if we can prove it